packages/spotlight/Spottable/Spottable.js

```diff
@@ -362,9 +345,13 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 
 		handleBlur = (ev) => {
 			if (this.shouldPreventBlur) return;
-
-			if (ev.currentTarget === ev.target) {
-				this.setState({focused: false, focusedWhenDisabled: false});
+			this.isFocused = false;
```

Comment:
We should move this down to be guarded against `ev.currentTarget === ev.target` since if a spottable contains a spottable (it's not a real typical scenario, but there is spotlight support for it) the focused state will be incorrectly set here. That will mean moving the `this.focusedWhenDisabled` to guard the `focusedWhenDisabled` value change and update - like in original `Spottable` implementation: https://github.com/enactjs/enact/blob/237836b72c49ceecd12fd9eda34ac81aae012395/packages/spotlight/Spottable/Spottable.js#L358